hellomama_registration/settings.py

```diff
@@ -237,6 +237,8 @@
     'registrations.created.sum',
     'registrations.created.total.last',
     'registrations.unique_operators.sum',
+    'change.language.sum',
+    'change.language.total.last'
```

Comment:
We confirmed with Jacqui that the current work is sufficient since the metrics will only be used to determine if the feature is being used.